src/game-engine.js

```diff
@@ -1,6 +1,7 @@
 
 import data from './data_wheel-of-fortune';
 import Round from './round';
+import Player from './player';
```

Comment:
My understanding was that it's best practice to capitalize the first letter on the file name as well as the class.  Maybe double check on that?